Composer/packages/client/src/recoilModel/dispatchers/publisher.ts

```diff
@@ -43,6 +44,7 @@ export const publishStorage = new ClientStorage(window.sessionStorage, 'publish'
 
 export const publisherDispatcher = () => {
   const publishFailure = async ({ set }: CallbackInterface, title: string, error, target, projectId: string) => {
+    TelemetryClient.track('PublishFailure', { message: error });
```

Comment:
Type of error here is unknown, L50 spreading suggests that it might be an object. If you pass it as string like right now, it will show up in the telemetry dashboard as [object Object].
My suggestion:
typeof error === 'string' ? error : JSON.stringify(error);

(you can always do JSON.stringify without the check for the type too but it would add extra double quotes around any string value)

Reply:
Ahh good catch, I have made this change